code/src/WijmoProvider/Features/Rows.ts

```diff
@@ -378,11 +377,19 @@ namespace WijmoProvider.Feature {
 
         // eslint-disable-next-line @typescript-eslint/no-explicit-any
         public getRowData(rowNumber: number): any {
-            return this._grid.isSingleEntity
-                ? OSFramework.Helper.Flatten(
-                      this._grid.provider.rows[rowNumber].dataItem
-                  )
-                : this._grid.provider.rows[rowNumber].dataItem;
+            let row: unknown;
```

Comment:
I think you don't need to create a variable first. You can directly it assign it on the try

Reply:
Already fixed